compiler/plc_xml/src/xml_parser/tests.rs

```diff
@@ -36,50 +35,151 @@ fn visit_and_desugar(content: &str) -> Result<Project, Vec<Diagnostic>> {
 
 #[test]
 fn variable_assignment() {
-    let pou = visit(content::ASSIGNMENT_A_B).unwrap();
+    // Replaces the `ASSIGNMENT_A_B` XML file
```

Comment:
Can we remove these "Replaces ..." comments? The files/strings they reference no longer exist, therefore I feel like they don't really need to be referenced here